src/main/menu.js

```diff
@@ -350,6 +333,17 @@ function getMenuTemplate () {
       click: () => dialog.openSeedFile()
     })
 
+    // Edit -> Preferencs (Windows, Linux)
```

Comment:
This comment should be "Edit menu (Windows, Linux)" to match the style of the others.